synapse/docker-compose.sytest.py37.redis.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3.1'
+
+services:
+
+  redis:
+    image: redis:5.0
+
+  sytest:
+    image: matrixdotorg/sytest-synapse:py37
+    depends_on:
+      - redis
+    env_file: docker-compose-env
+    environment:
+      POSTGRES: "1"
+      WORKERS: "1"
+      BLACKLIST: "synapse-blacklist-with-workers"
+      REDIS: "redis"
+    always-pull: true
+    workdir: "/src"
+    entrypoint: "/bin/sh"
+    init: false
+    shell: ["-x", "-c"]
+    mount-buildkite-agent: false
```

Comment:
(you don't need this with the docker-compose buildkite plugin, because it doesn't suffer the same foot-gun as the docker buildkite plugin where failing to set it leaks the buildkite agent token into the container. See also https://github.com/matrix-org/internal-config/wiki/BuildKite#not-running-untrusted-code-on-the-build-agents.)